backend/typescript/models/volunteer.model.ts

```diff
@@ -0,0 +1,14 @@
+import { Column, DataType, Model, Table, ForeignKey, BelongsTo, AllowNull } from "sequelize-typescript";
+
+import User from "./user.model"
+
+@Table({ tableName: "volunteers" })
+export default class Volunteer extends Model {
+    @ForeignKey(() => User)
+    @AllowNull(false)
+    @Column({ type: DataType.INTEGER })
+    user_id!: number;
+
+    @BelongsTo(() => User)
+    user!: User;
```

Comment:
This should reference the UserDTO instead of the User. You can import it from the types file. Can you also please create a volunteerDTO in the types file as well?